### PR TITLE
wip: complex example showcasing multiple plugins

### PR DESCRIPTION
**Pull request by n1ru4l**

### CODE REVIEW

File: examples/complex/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# TODO
```
Comment (dotansimha) [inline examples/complex/README.md:1]: TODO!

**Review by dotansimha — APPROVED** [review]
Looks great :) I guess just the README is missing? or is there something else?

Also, for the Auth0 plugin, maybe it's a good chance to update the README and make sure it's updated?

File: examples/complex/README.md (hunk 1 above)
Comment (n1ru4l) [reply examples/complex/README.md:1]: Ah it is not done yet

**Comment by n1ru4l** [comment]
Gonna close this for now as I am not really sure where I wanted this to go.

**Pull request closed without merging**